recipes/ibmz_ci.rb

```diff
@@ -68,11 +68,11 @@ class ::Chef::Recipe
   email-ext:2.62
```

Comment:
Since we updated the `cloud-stats` plugin to 0.14 in the `powerci` recipe, should this be updated as well?

Reply:
It's not required so I'd rather not.